baton-maven-plugin/src/test/resources/specifications/baton-maven-plugin.feature

```diff
@@ -1,12 +1,64 @@
+@baton
 Feature: Validate the definition of a migration target
 
-  Scenario: A simple migration can be defined and executed
-    Given a migration target with a name and a implementation
+  Scenario: Group migrations can be defined and executed in their order
+    Given a migration file 
+        """
+        [ 
```

Comment:
S: normally, we are happy to hide some of the detail here.  
```
Given a migration files with <important part for this scenario>" (dirty work or mapping to a file in the step)
...

// So here, maybe something like:
Given a mogration with groups in the order of A, B, C
```

For this test, the only thing the reader cares about is the order of the groups.  But they have to read through the rest of it before realizing that.  We want to be focused instead.

Reply:
Test has been modified to focus on the group ordering.